### ISSUE TITLE
Stop typer automatically

### ISSUE THREAD

**Issue by ranggasan** [straversi/Typer.js#8]
Is there any way to stop typer after sentence has completed?

**Comment by straversi** [comment]
No, but great idea. I'll keep this issue open until it is implemented.

**Comment by shubdev07** [comment]
please introduce this functionality

**Comment by straversi** [comment]
@shubdev07 could you give an example of how you would use it? I’d love to add this, and I want to make sure I understand the case.
For example, do you have one word you want typed out? Or would you like it to stop after typing and deleting several words?

**Comment by andrebrandao-dev** [comment]
i will appreciate it, if somehow we can stop typer with my final word `data-words="Design, Create, Build, Planner"` maybe with `data-end="Build"` to a specific word or just `data-end="true"` to stop with my final word

**Comment by straversi** [comment]
@andrebrandaoo There is a `data-loop` count already that can be set to `"1"`. So we can achieve the same effect by adding `data-delete-last="false"` or something to that effect. So to make `data-word="design,create,build"` stop on "build" without deleting, we could write:
```
... data-loop="1" data-delete-last="false" ...
```
`data-delete-last="false" ` could even be the default functionality, as it makes more sense from a UI perspective.
What do you think?

**Comment by andrebrandaoo** [comment]
@straversi it will be nice too

**Comment by rehashedprime** [comment]
I have reviewed the typer.js file & found no data-loop reference im trying to implement typer.js to give a terminal like feeling to a webpage but i break it everytime i try to get rid of the delete delay function.. any insight on how i can use typer.js without having the text deleted after being typed

**Comment by amirfahimnia** [comment]
@straversi Any idea when you think you can implement this option?

**Comment by straversi** [comment]
👋 This is implemented now! `data-loop="1"` will type out all of the words in the list, then stop _without_ deleting the last word.
